.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true;
```

Comment:
Do we want an .editorconfig in the master repo?

Reply:
Sorry. Probably not. That was more a formatting issue I was trying to fix on my end. Steven can probably just ignore that. I should have added that to the .gitignore but wasn't sure it git would then ignore my formatting.

Reply:
Shouldn't ignore your formatting if it's local still. Pretty sure most of these should be covered in .eslintrc, right? Then all your formatting will transfer to us and it'll become standard.